### PR TITLE
Crimson Cross Arsenal

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
A new bluecross shield that sprited by gid_git
The  bluecross shield based on *direct kills* grows stronger both in health and at key milestones damage/added armor
One of the quirks is that it protects *every* lim no matter what and its block odds start out insainly low (-15%)

A new crimson cross weapon Vail Render
Deals damage based on how hurt you are  + how hurt your target is
in PVP or against things with negative health that should be dead at a lesser rate

Converts the Oni Club into a crimson cross weapon
Deals burn damage
Attack DELAY builds the more damage you are
Increases AP of the weapon based on how hurt you are

### CODE REVIEW

**Comment by benj8560** [comment]
no on body sprite? Little important theres a visual aid for seeing someone has a shield.

File: code/modules/projectiles/guns/oddity_items.dm [hunk 1]
```diff
@@ -537,6 +542,98 @@
...
 	return TRUE //We are always adjacent
 
+// Shield
+
+/obj/item/shield/riot/mass_grave
+	name = "grave marker shield"
+	desc = "An anomalous weapon created by an unknown person (or group?), their work marked by a blue cross, these items are known to vanish and reappear when left alone. \
+			A large grave marker degraded by the sands of time to be unreadable. This shield will always help block any physical attack. Has the power of endless growth in power the more *direct* kills directly caused by the shield."
+	icon_state = "mass_grave" //SOB MAKING SPRITES IS SO HARD - Trilby
+	item_state = "tray_walk"
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/oddity_items.dm:552]:
```suggestion
	item_state = "tray"
```

File: code/modules/projectiles/guns/oddity_items.dm [hunk 2]
```diff
@@ -537,6 +542,98 @@
...
 	return TRUE //We are always adjacent
 
+// Shield
+
+/obj/item/shield/riot/mass_grave
+	name = "grave marker shield"
+	desc = "An anomalous weapon created by an unknown person (or group?), their work marked by a blue cross, these items are known to vanish and reappear when left alone. \
+			A large grave marker degraded by the sands of time to be unreadable. This shield will always help block any physical attack. Has the power of endless growth in power the more *direct* kills directly caused by the shield."
+	icon_state = "mass_grave" //SOB MAKING SPRITES IS SO HARD - Trilby
+	item_state = "tray"
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/oddity_items.dm:552]:
```suggestion
	item_state = "tray_shield"
```

**Comment by AysheDaArtist** [comment]
I'm working on a sprite for this!  Have it here within the next 24 hours

**Pull request merged**